### PR TITLE
adding terraform, updating pipelines page

### PR DESCRIPTION
**Pull request by Dosplatos**

### CODE REVIEW

File: apps/docs/src/pages/en/guides/pipelines.mdx [hunk 1]
```diff
@@ -21,12 +22,14 @@ are what you will be attaching pipelines to.
 
 ## What are Pipelines?
 
-Pipelines are logical checks and rules data must pass through and clear so it 
-can continue to be processed by your applications, and subsequently sent 
-"downstream" (i.e. written to a database, or passed to another application or 
-service). Pipelines are put together in the form of steps (which we refer 
-to interchangeably as rules), and each step has a type which determine the 
-kinds of actions to take on data.
+In simple words, it's a workflow for data.
```
Comment (dselans) [inline apps/docs/src/pages/en/guides/pipelines.mdx:25]: "Pipelines in Streamdal work similarly to pipelines in other products that utilize step-based workflows."

File: apps/docs/src/pages/en/guides/terraform.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Terraform
+metaTitle: "Streamdal Terraform Provider"
+description: Information on Streamdal Terraform Provider
+layout: ../../../layouts/MainLayout.astro
+---
+
+Alternatively to defining pipelines via the UI, and to enhance pipeline 
+portability, we offer a [Terraform provider](https://registry.terraform.io/providers/streamdal/streamdal/latest). 
+This allows you to interact with the Streamdal Server API
```
Comment (dselans) [inline apps/docs/src/pages/en/guides/terraform.mdx:10]: This allows you to define pipelines, steps, audiences and assignments programmatically.

**Review by dselans — APPROVED** [review]
Minor copy updates

File: apps/docs/src/pages/en/guides/pipelines.mdx (hunk 1 above)
Comment (Dosplatos) [reply apps/docs/src/pages/en/guides/pipelines.mdx]: Updated

File: apps/docs/src/pages/en/guides/terraform.mdx (hunk 2 above)
Comment (Dosplatos) [reply apps/docs/src/pages/en/guides/terraform.mdx]: Updated

**Pull request merged**